mintpy/ifgram_inversion.py

```diff
@@ -189,6 +189,9 @@ def cmd_line_parse(iargs=None):
         elif inps.obsDatasetName.startswith('rangeOffset'):
             inps.outfile = ['timeseriesRg.h5', 'residualInvRg.h5', 'numInvOffset.h5']
 
+        elif inps.obsDatasetName.startswith('ion'):
+            inps.outfile = ['timeseriesIon.h5', 'residualInvIon.h5', 'numInvIon.h5']
+
```

Comment:
for ionosphere, it's still phase, so we should use `temporalCoherenceIon.h5` instead of `residualInvIon.h5`.

Reply:
I see. I will correct that. Thanks.